### DISCUSSION TITLE
Bug: Error [VOICE_CONNECTION_TIMEOUT]: Connection not established within 15 seconds.

### DISCUSSION THREAD

**Discussion by dickyshndri** [eritislami/evobot#368] [Q&A]
Error [VOICE_CONNECTION_TIMEOUT]: Connection not established within 15 seconds.

**Comment by dylanjamesdev** [comment]
This is a timeout on discord’s end, usually due to poor internet. You can change the amount of seconds it takes to timeout in your client options.

**Comment by eritislami** [comment]
Not a bug, i will convert to discussion
